### ISSUE TITLE
Fallback UI throws React error and does not render fallback UI

### ISSUE THREAD

**Issue by hsdonkin** [rollbar/rollbar-react#19]
As titled, Fallback UI does not render using `fallbackUI` prop on `<ErrorBoundary />` component.

Using React 17.0.2 and Next.js 11.1.0.

```
The tag <fallbackUI> is unrecognized in this browser.
If you meant to render a React component, start its name with an uppercase letter.
```

**Comment by bondom** [comment]
@hsdonkin I got the same error when I tried to pass rendered react component in fallbackUI prop, which seems to be incorrect:
```
<ErrorBoundary
                    level={LEVEL_ERROR}
                    fallbackUI={<MyComponent />}
```

Instead try to pass just name of your component, or use function:
```
<ErrorBoundary
                    level={LEVEL_ERROR}
                    fallbackUI={MyComponent}
```
or 
```
<ErrorBoundary
                    level={LEVEL_ERROR}
                    fallbackUI={() => (<MyComponent />)}
```
```

**Comment by waltjones** [comment]
This is now published in v0.10.0. https://github.com/rollbar/rollbar-react/releases/tag/v0.10.0
